site/content/events/2018-09-21.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: Techstars Startup Weekend NYC | Latinx in Tech
 date: 2018-09-21
-image: https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F49199101%2F1954445013%2F1%2Foriginal.jpg?w=800&auto=compress&rect=0%2C0%2C2160%2C1080&s=fb734455c7d0e7ed914696216b71b50c
```

Comment:
Why no image now? 😭

Reply:
The images don't look well on the color thing, we'd have to make a calculation on the background too. Coz the image is blue and text is white etc :( 

We can create a ticket for the future